ten_quick_tips_live_coding.md

```diff
@@ -152,58 +154,39 @@ and these may help learners to understand the stages of the lesson and to organi
 What can work really well is when you as instructor generate the illustrations on the whiteboard as you progress through the material.
 This allows you to build up diagrams,
 making them increasingly complex in parallel with the material you are teaching.
-[FIXME: mention dual coding and the value of presenting complementary information on visual and linguistic channels.]
+Presenting complementary information using visual and verbal representations helps learning (so-called "dual coding") (Clark, J.M. & Paivio, A. Educ Psychol Rev (1991) 3: 149. https://doi.org/10.1007/BF01320076).
 
 Diagramming helps learners understand the material,
-makes for a more lively workshop (you'll have to move between your computer and the blackboard),
+makes for a more lively workshop (you'll have to move between your computer and the whiteboard),
 and gathers the learners' attention to you as well.
-[FIXME: more detail here?]
 
 ## Tip 7. Stick to the lesson material.
 
-Participatory live coding when teaching works best with a well-developed and tested script.
-[FIXME: GVW doesn't like the term "script". Worth making the analogy with jazz - there's a score, but you're expected to improvise on top of it?]
+Participatory live coding when teaching works best with a well-developed and tested lesson.
```

Comment:
I disagree with this claim: I think live coding is much better than all the alternatives when you _don't_ have a well-developed lesson, but maybe that's me.

Reply:
Adjusted to "When getting started with participatory live coding when teaching, it is advised to use a well-developed and tested lesson."